Source/WebCore/platform/graphics/cairo/CairoUtilities.cpp

```diff
@@ -49,13 +49,17 @@
 
 namespace WebCore {
 
-#if USE(CAIRO) && !PLATFORM(GTK)
+static auto s_defaultCairoFontOptions = CairoUniquePtr<cairo_font_options_t>(cairo_font_options_create());
```

Comment:
Could we avoid the initialization here? In the case of GTK we avoid one allocation, since we always call setDefaultCairoFontOptions().